chipsec/hal/uefi_platform.py

```diff
@@ -37,7 +37,7 @@
 from chipsec.hal.uefi_common import EFI_VARIABLE_HARDWARE_ERROR_RECORD, EFI_VARIABLE_AUTHENTICATED_WRITE_ACCESS, EFI_VARIABLE_TIME_BASED_AUTHENTICATED_WRITE_ACCESS
 from chipsec.hal.uefi_fv import NextFwVolume, NextFwFile, EFI_FVB2_ERASE_POLARITY, EFI_FV_FILETYPE_RAW
 
-EfiVariableType = Tuple[int, Optional[bytes], Optional['EFI_TABLE_HEADER'], bytes, str, int]
+EfiVariableType = Tuple[int, bytes, 'EFI_TABLE_HEADER', bytes, str, int]
```

Comment:
Hello, the `Optional['EFI_TABLE_HEADER']` could be kept, as `getEFIvariables_NVAR` can return variables with `(nof, b'', None, data, guid, attribs)`.

By the way, while trying to add type hints to `chipsec/hal/uefi_platform.py`, I ended up using a `Union` for the header type:

```python
EfiVariableType = Tuple[int, bytes, Union["EFI_TABLE_HEADER", "EFI_HDR_VSS", "EFI_HDR_VSS_AUTH", "EFI_HDR_VSS_APPLE", None], bytes, str, int]
```

This matches the return type of `_getEFIvariables_VSS` better. What do you think?